Cargo.lock

```diff
@@ -3084,6 +3171,7 @@ version = "4.2.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "c3dfe1b7eb6f9dcf011bd6fad169cdeaae75eda0d61b1a99a3f015b41b0cae39"
 dependencies = [
+ "chrono",
```

Comment:
Issue: Where did `chrono` come from; we already have `time` in here. I'd prefer to keep just that one big time library in Cargo.lock

Reply:
probably from the `lru` dep in `md-tests`?